upload/File.php

```diff
@@ -1,39 +1,40 @@
 <?php
+
 namespace upload;
 
 class File extends Validate
 {
-
-  function __construct($key, $File = false)
-  {
-    if (!$File) {
-      $File = array(
-        'name' => $_FILES[$key]['name'],
-        'tmp_name' => $_FILES[$key]['tmp_name']
-      );
+    public function __construct($key, $File = false)
```

Comment:
__[phpmd]__
* The method __construct has a boolean flag argument $File, which is a certain sign of a Single Responsibility Principle violation.
* __construct accesses the super-global variable $_FILES.
* The parameter $File is not named in camelCase.
* The variable $File is not named in camelCase.
* The variable $FilePath is not named in camelCase.
* The variable $FileName is not named in camelCase.

<!-- Comment by [SideCI](https://sideci.com) -->